tests/admin_views/tests.py

```diff
@@ -911,7 +911,11 @@ def test_isnull_lookups(self):
 
     def test_logout_and_password_change_URLs(self):
         response = self.client.get(reverse("admin:admin_views_article_changelist"))
-        self.assertContains(response, '<a href="%s">' % reverse("admin:logout"))
+        self.assertContains(
```

Comment:
Should we set `html=True` here?

Reply:
It won't work with `html=True` because we don't compare the entire `form`, and response doesn't contain:
```html
<form id="logout-form" method="post" action="/test_admin/admin/logout/"></form>
```